Dockerfile

```
@@ -1,8 +1,8 @@
 ARG UBUNTU_VERSION=20.04
 FROM ubuntu:${UBUNTU_VERSION} as haskell-builder
 ARG CABAL_VERSION=3.2.0.0
-ARG CARDANO_NODE_VERSION=1.27.0
-ARG CARDANO_DB_SYNC_VERSION=10.0.1
+ARG CARDANO_NODE_VERSION=alonzo-white-1.4
```

```suggestion
ARG CARDANO_NODE_VERSION=alonzo-purple-1.0
```
The target tag is now `alonzo-purple-1.0`. Config for _alonzo-qa_ network is in https://github.com/input-output-hk/cardano-configurations/commit/ad34c4ef391124d4582e1bf22307950fa299f26b

---

Is necessary to recreate the snapshots against that tag too?

---

It should be fine for this preliminary version, as the `cardano-db-sync` version has not changed.